readme.md

```diff
@@ -37,6 +39,7 @@ RoadRunner is an open-source (MIT licensed), high-performance PHP application se
     * [Building a Server](beep-beep/build.md)
     * [RPC](beep-beep/rpc.md)
     * [Write a Plugin](beep-beep/plugin.md)
+    * [Service plugin](beep-beep/service.md)
```

Comment:
Running External Services

Reply:
@wolfy-j This is a confusing concept (at least for me). It's not about `external` services. It's about running its own processes as services. Also, I guess, that simpler for a user to find `Service` plugin in the docs, than `Running external services` which is not clear, what plugin is responsible for that.